SparkConnector/sparkconnector/configuration.py

```diff
@@ -399,15 +399,12 @@ def get_spark_session_config(self):
                                               '&var-UserName=' + self.get_spark_user() + \
                                               '&var-ApplicationId=' + app_id
 
-            # determine the history server URL depending on the selected resource manager (yarn, k8s, local etc)
-            historyserver_url = self._get_sc_config(
-                'spark.yarn.historyServer.address',
+            # determine the WebUI URL for Spark on YARN
```

Comment:
Can we add here in the comment that what we obtain is the URLs of the Hadoop NNs?